io.github.nemuelw.protodesk.yml

```diff
@@ -0,0 +1,40 @@
+app-id: io.github.nemuelw.protodesk
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+base: io.qt.PySide.BaseApp
+base-version: '6.7'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: protodesk
+
+finish-args:
+  - --device=dri
+  - --filesystem=host
```

Comment:
why is host needed? Does it not use filechooser portals?

Reply:
It sure does. In choosing to add host I hadn't considered that filechooser portals is enough for the application's needs.